### ISSUE TITLE
Device ACL support

### ISSUE THREAD

**Issue by yshui** [ConsoleKit2/ConsoleKit2#105]
systemd-logind will give access of devices tagged with 'uaccess' to the user of the active seat. 

Can consolekit2 do the same thing?

**Comment by 375gnu** [comment]
Yes, but in CK2 the tag is called 'udev-acl', build it with `--enable-libudev --enable-udev-acl`

**Comment by yshui** [comment]
@375gnu I see. I thought it didn't. But now I found out it just doesn't work with hotplugged devices.

I have to switch to another vt and switch back to have the permissions applied. Is this a bug?

**Comment by 375gnu** [comment]
@yshui could you give a bit more info? what devices, what permissions were applied on vt switch, i'll check on my pc if i can reproduce your issue.

**Comment by yshui** [comment]
@375gnu I have an Yubikey NEO. When inserted, the dev file is owned by root, with permission 600. `udevadm info` shows it has the udev-acl tag. Then I switched to another vt (Ctrl-Shift-F*) and switch back. And the dev file will gain user rw permissions (660).

**Comment by yshui** [comment]
I just found out I was somehow running elogind and consolekit at the same time.

The behavior I described seems to be elogind's. After I removed elogind, the ACL is not applied _at all_
